examples/ntfs_crawler.rs

```diff
@@ -30,16 +30,30 @@ where
 }
 
 fn run() -> Result<()> {
-    let objects = transit_model::ntfs::read(".")?;
+    // load ntfs from current directory
+    let transit_objects = transit_model::ntfs::read(".")?;
 
-    for (from, stop_area) in &objects.stop_areas {
-        let cms = get(from, &objects.commercial_modes, &objects);
-        let pms = get(from, &objects.physical_modes, &objects);
-        let ns = get(from, &objects.networks, &objects);
-        let cs = get(from, &objects.contributors, &objects);
+    // stop_area by stop_area, print PT objects related to it
+    for (idx, stop_area) in &transit_objects.stop_areas {
+        // retrieve idx from id
+        assert_eq!(
+            transit_objects.stop_areas.get_idx(&stop_area.id).unwrap(),
+            idx
+        );
+
+        // lines passing by stop
+        let lines = get(idx, &transit_objects.lines, &transit_objects);
```

Comment:
completly minor, but don't you think that for the purpose of an example it would be more straightforward to directly use `get_corresponding`? (at least for the first ones)

Reply:
True (especially if it goes in main readme), I tried to mix existing and suggestions :dancers: